charts/kafka-operator/templates/operator-kafka-crd.yaml

```diff
@@ -1,20 +1,21 @@
+
+---
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  labels:
-    app.kubernetes.io/name: {{ include "kafka-operator.name" . }}
```

Comment:
These helm specific labels shouldn't be removed. Unfortunately the gen tool used by kubebuilder is not helm aware thus we need to maintain these labels manually to make the helm charts to be proper

Reply:
Do you feel strong for keeping helm labels in crd @stoader?
The reason I am asking is that we also have an internal patch to move to helm3 where crds are not managed as templates anymore. 

https://geeksocket.in/posts/helm-2-3-migration/ goes at lenght on explaining this. Basically to keep compat between helm2 amd helm3 helm is recommending to move out crds in a separate dir in the chart

Reply:
I am talking specifically about this change https://github.com/adobe/kafka-operator/commit/9537fce209934f9fed9076a3d36c22288a40a303 we wanna propose

Reply:
Thinking it more I think you're right.